### PR TITLE
dev to main for beta2

### PR DESCRIPTION
**Pull request by shalehaha**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by nporter-adbe — APPROVED** [review]
looks good.

@emdobrin are we ready to merge dev to main?

**Review by kevinlind — COMMENTED** [review]
Let's update `SampleApps/AEPCommerceDemoApp/Podfile.lock` with the latest as well.

**Review by kevinlind — APPROVED** [review]
Looks good to me too.

File: AEPEdge.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1554,7 +1568,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				MARKETING_VERSION = "1.0.0-beta-1";
+				MARKETING_VERSION = "1.0.0-beta-2";
```
Comment (emdobrin) [inline AEPEdge.xcodeproj/project.pbxproj:1571]: I know we decided to change the version format to something like`1.0.0-beta.2` and we adopted this standard in the core extensions, we should update the Edge dev too before releasing.

**Review by emdobrin — APPROVED** [review]

**Pull request merged**